.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```
@@ -158,7 +158,8 @@ jobs:
           }
 
           if is_vanilla_build; then
-            echo "apt-get update && apt-get install -y qemu-user gdb" | docker exec -u root -i "$id" bash
+            echo "apt-get update && apt-get install libgnutls30" | docker exec -u root -i "$id" bash
```

Use `||` to ignore cert failure during install
```suggestion
            echo "apt-get update || apt-get install libgnutls30" | docker exec -u root -i "$id" bash
```